.circleci/config.yml

```diff
@@ -35,7 +35,7 @@ executors:
                 description: "Which cimg/postgres tag to use"
                 type: string
         docker:
-            - image: cimg/node:20.9
+            - image: cimg/node:20.10
```

Comment:
nit: why specific `20.10.0` above on L6 and L13, but drop the `.0` patch version here?
Although minorest of nits since it looks like it was that way before too.

Reply:
Yeah, the only reason I did it was because that's how we did it before - I don't know the original reason. Personally I think it's pretty safe to leave off the patch version, for places that Dependabot can't update for us.